auth0/v3/management/device_credentials.py

```diff
@@ -34,7 +34,7 @@ def _url(self, id=None):
             return '{}/{}'.format(url, id)
         return url
 
-    def get(self, user_id, client_id, type, fields=None, include_fields=True):
+    def get(self, user_id, client_id, type, page=None, per_page=None, fields=None, include_fields=True):
```

Comment:
With the new parameters added here, wouldn't existing calls to this method using positional parameters break?